### PR TITLE
Create form

### PR DESCRIPTION
**Pull request by annadragan**

### CODE REVIEW

File: src/pages/VocabularyCards.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import CardList from '../components/CardList';
+import Header from '../components/Header';
+
+export default function VocabularyCards({ vocabulary }) {
+  return (
+    <>
+      <Header>Vocabulary Cards</Header>
+      <CardList vocabulary={vocabulary} />
+      <Link to={'/form'}>
+        {' '}
```
Comment (HannaSophia-92) [inline src/pages/VocabularyCards.js:13]: Delete line 13 :)

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import styled, { css } from 'styled-components';
+
+export default function Button() {
+  return (
+    <ButtonStyled aria-label="button" role="button">
+      Create a card
+    </ButtonStyled>
+  );
+}
```
Comment (HannaSophia-92) [inline src/components/Button.js:9]: I think you don´t really need the function here. You could do: 
```suggestion
export default styled.button`
....
`
```
...directly and import the Button component where you need it :)

File: src/components/CardList.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
-import { vocabulary } from '../db';
+import Card from './Card';
 
-export default function CardList() {
+export default function CardList({ vocabulary }) {
   return (
     <>
       <ListWrapper role="list" aria-label="vocabulary cards" rows={3}>
-        {vocabulary.map(({ _id, word, example, explanation }) => {
+        {vocabulary?.map(({ _id, word, example, explanation }) => {
```
Comment (HannaSophia-92) [inline src/components/CardList.js:8]: Delete the questionmark "?"

File: src/components/Form.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+      <FormWrapper
+        aria-describedby="Create a new word"
+        onSubmit={handleSubmitTerm}
+        autoComplete="off"
+      >
+        <Header>Card form</Header>
+        <InputWrapper>
+          <Input
+            labelText="New word:"
+            textarea
```
Comment (HannaSophia-92) [inline src/components/Form.js:27]: Do you need the "textarea" here?

File: src/components/Form.spec.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router-dom';
+import Form from './Form';
+
+describe('Form', () => {
+  it('renders a form with three inputs and a submit button, does not submit wenn the input is empty or wenn there are only whitespaces and it has a title', () => {
```
Comment (HannaSophia-92) [inline src/components/Form.spec.js:7]: A little typo = "does not submit WHEN..." :)

**Review by HannaSophia-92 — COMMENTED** [review]
Nice work Anna :) I have only minor issues to think about 🚀

File: src/components/Form.spec.js (hunk 5 above)
Comment (annadragan) [reply src/components/Form.spec.js]: thank you! :)

**Pull request merged**